kubecost.yaml

```diff
@@ -20317,28 +20303,6 @@ subjects:
     name: kubecost-cost-analyzer
     namespace: kubecost
 ---
-# Source: cost-analyzer/templates/cost-analyzer-psp-rolebinding.template.yaml
-apiVersion: rbac.authorization.k8s.io/v1
-kind: RoleBinding
-metadata:
-    name: kubecost-cost-analyzer-psp
-    namespace: kubecost
-    labels:
-
-      app.kubernetes.io/name: cost-analyzer
-      helm.sh/chart: cost-analyzer-1.101.3
-      app.kubernetes.io/instance: kubecost
-      app.kubernetes.io/managed-by: Helm
-      app: cost-analyzer
-roleRef:
-    apiGroup: rbac.authorization.k8s.io
-    kind: Role
-    name: kubecost-cost-analyzer-psp
-subjects:
-- kind: ServiceAccount
-  name: kubecost-cost-analyzer
-  namespace: kubecost
----
```

Comment:
Does the helm chart also remove these or are we manually removing from the rendered content?

Reply:
this will not remove. users would need to manually remove.

just to be clear- this file is an alternative to helm.